docs/user-guide.md

```diff
@@ -780,24 +794,24 @@ need a different value. The parameter values that the toolkit uses are echoed
 during execution so you can confirm the configuration.
 
 The complete list of parameters and their values are provided in the [Parameters
-section](#heading=h.m1e7sxtn4kzm).
+section](#parametersn).
 
 Configuring an installation is discussed in [Configuring
```

Comment:
I know you didn't change this, but I think this sentence and its link should be deleted, because it is kind of self referential.